net/acme-common/files/acme.init

```diff
@@ -104,14 +110,16 @@ get_cert() {
 	fi
 
 	load_credentials() {
-		eval export "$1"
+		eval procd_append_param env "$1"
 	}
 	config_list_foreach "$section" credentials load_credentials
 
-	"$HOOK" get
+	procd_close_instance
```

Comment:
This is never called if we return early above. Which, AFAICT, leaves a half-open instance definition for when the function is called again (assuming there are multiple sections configured). I doubt that will work correctly?